test/unit/specs/components/staking/TabMyDelegations.spec.js

```diff
@@ -17,7 +17,8 @@ const getters = {
   committedDelegations: {
   },
   connected: true,
-  bondDenom: `stake`
+  bondDenom: `stake`,
+  session: { signedIn: true }
```

Comment:
we should add a snapshot when not signed in

Reply:
Done. tests should move into TmPage (which is barely tested) https://github.com/cosmos/voyager/issues/2156